client/src/App.tsx

```diff
@@ -25,19 +25,27 @@ import { Payload } from './lib/payload';
 import { SearchIcon } from '@chakra-ui/icons';
 import { search } from './lib/search';
 
+function debounce(f: (v: string) => void, delay: number) {
```

Comment:
Small nitpick: move this into `App` and make it an arrow function just to stay consistent with what we already have.

Reply:
I didn't do this because debounce could potentially be used in other places and moving it into `App` would cause a new copy of it to be generated in every render.

Reply:
Ah good point, I'd then just make it an arrow function and run `prettier --write .`, then this should be good to merge if ci passes